packages/mobile/src/identity/revoke.ts

```diff
@@ -31,11 +31,11 @@ export function* revokePhoneMapping(e164Number: string, account: string) {
       phoneHash
     )
 
-    if (status.total < 1) {
-      throw new Error('No attestations associated with identifier')
+    if (status.isVerified) {
```

Comment:
If only typing worked for saga results, we could avoid whole classes of bugs like these.